src/Microsoft.TemplateEngine.Cli/Commands/list/BaseListCommand.cs

```diff
@@ -41,6 +42,11 @@ internal class BaseListCommand : BaseCommand<ListCommandArgs>, IFilterableComman
 
         public IReadOnlyDictionary<FilterOptionDefinition, Option> Filters { get; protected set; }
 
+        internal static Option<bool> IgnoreConstraintsOption { get; } = new("--ignore-constraints")
```

Comment:
Consider making arity Zero here to prevent ambiguity in the grammar - since the implicit default is `false` if not present and `true` if present, the UX of allowing a boolean after the flag is less than ideal. This is something I'm trying to unify across the SDK where possible.

Reply:
I think this behavior is available by default in `System.CommandLine`, but might be wrong.

Reply:
Added